<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6NQ8E1M" failures="2" tests="3" name="Test" time="13.091" errors="0" timestamp="2025-06-24T00:14:04 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.144">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at com.stepcampus.automation.tests.stepdefs.LoginSteps.user_should_land_on_home_page(LoginSteps.java:28)
at ✽.User should land on Home Page(classpath:features/login.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.161">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at com.stepcampus.automation.tests.stepdefs.LoginSteps.user_should_land_on_home_page(LoginSteps.java:28)
at ✽.User should land on Home Page(classpath:features/login.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="11.71"/>
</testsuite> <!-- Test -->
